src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -144,9 +143,13 @@ var inputValueTracking = {
     }
 
     var lastValue = tracker.getValue();
-    var nextValue = getValueFromNode(
-      ReactDOMComponentTree.getNodeFromInstance(inst),
-    );
+
+    var node = inst;
+    if (!(inst: any).nodeName) {
```

Comment:
Let's use same check as in [here](https://github.com/facebook/react/blob/cff012fc16ea6e7f40398410aacc3be20dc39804/src/renderers/dom/shared/findDOMNode.js#L57) and leave a TODO to remove it when Stack is removed.